notebooks/monitoring.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+
+declare -a TEMP=$(mktemp /temp_monitoring.XXXXXXXX)
+
+
+function get_disk_info() {
+        # df command and cromwell root field
+        disk_info=$(df | grep cromwell_root)
+        echo "$disk_info"
+}
+
+
+function get_disk_usage() {
+        # get disk usage field
+        echo $(get_disk_info | awk '{ print $5 }')
+}
+
+
+function get_mem_info() {
```

Comment:
This one is probably overkill... should be OK to inline `cat /proc/meminfo` since it's just one command